backends/core/models.py

```diff
@@ -58,7 +67,15 @@ class Pipeline(BaseModel):
   params = relationship('Param', lazy='dynamic', order_by='asc(Param.name)')
 
   def __init__(self, name=None):
+    self.memcache_client = memcache.Client()
     self.name = name
+    cached_values = {
+      CACHE_KEY_FAILED_JOBS: 0, 
+      CACHE_KEY_REMAINING_JOBS: len(self.jobs.all()), 
+      CACHE_KEY_LIST_OF_TASKS_ENQUEUED: []
+    }
+    self._add_or_update_multi_to_cache(cached_values)
```

Comment:
Can you call `self.get_ready()` here to be [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)?

Reply:
Or maybe, juste remove the logic, since in the Job model we don't initialize the variables like this.
Feel free to pick and choose the option you prefer.

Reply:
done